src/sectionproperties/pre/library/timber_sections.py

```diff
@@ -0,0 +1,66 @@
+"""Timber sections library."""
+
+from __future__ import annotations
+
+import sectionproperties.pre.geometry as geometry
+import sectionproperties.pre.library.primitive_sections as primitive_sections
+import sectionproperties.pre.pre as pre
+
+
+def clt_rectangular_section(
+    d: list[float],
+    lay_orient: list[int],
+    b: float,
+    timb_mat0: pre.Material = pre.DEFAULT_MATERIAL,
+    timb_mat90: pre.Material = pre.DEFAULT_MATERIAL,
+) -> geometry.CompoundGeometry:
+    """Constructs a timber rectangular section.
+
+    Constructs a timber rectangular section of depth ``d`` and width ``b``.
+
+    .. note::
+
+    Args:
+        d: Timber layer section thickness
+        lay_orient: List of layer orientation
```

Comment:
I think `lay_orient` needs a little more explanation. The implementation of this function assumes certain values to be in there, namely `0` and any other value. The lay orientation could just as well be `[0, 1, 0, 1, 0]` instead of `[0, 90, 0, 90, 0]` and the function would work the same (it could also use `[0, 34, 0, 
-15, 0,]` and you would get the same result).

I think a list of values to describe the layup makes sense but it seems to rely, somewhat, on "magic values" that are not explained. I think it would be good to say something like:

`lay_orient: A list of layer orientations, e.g. [0, 90, 0, 90, 0], where 0 indicates where `timb_mat0` is to be used and where 90 indicates where `timb_mat1` is to be used.`